src/material-experimental/mat-feature-highlight/feature-highlight-callout-container.spec.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentPortal, PortalModule, TemplatePortal} from '@angular/cdk/portal';
+import {Component, NgModule, TemplateRef, ViewChild} from '@angular/core';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {async} from '@angular/core/testing';
+import {By} from '@angular/platform-browser';
+
+import {FeatureHighlightCalloutContainer} from './feature-highlight-callout-container';
+
+describe('FeatureHighlightCalloutContainer', () => {
+  let fixture: ComponentFixture<CalloutHostComponent>;
+  let comp: CalloutHostComponent;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        CalloutContainerTestModule,
+      ],
+    });
+
+    TestBed.compileComponents();
+    fixture = TestBed.createComponent(CalloutHostComponent);
```

Comment:
Can you switch the tests so each individual one creates the component itself? Creating it like this in a `beforeEach` makes it difficult to add new tests that use a different template. You can see an example in `drag.spec.ts`.